dgr/common/dgr-manifest.go

```diff
@@ -74,6 +74,8 @@ type BuilderDefinition struct {
 
 type BuildDefinition struct {
 	MountPoints []MountInfo `json:"mountPoints,omitempty" yaml:"mountPoints,omitempty"`
+	Exclude     []string    `json:"exclude,omitempty" yaml:"exclude,omitempty"`
```

Comment:
exclude & transform is not linked to reproductible build, can you put it in a separate PR please ?